internal/controller/model/transform_types.go

```diff
@@ -50,10 +50,8 @@ const (
 )
 
 const (
-	AppInstanceLabelKey            = "app.kubernetes.io/instance"
-	KBManagedByKey                 = "apps.kubeblocks.io/managed-by"
-	RoleLabelKey                   = "kubeblocks.io/role"
-	ConsensusSetAccessModeLabelKey = "cs.apps.kubeblocks.io/access-mode"
+	AppInstanceLabelKey = "app.kubernetes.io/instance"
+	KBManagedByKey      = "apps.kubeblocks.io/managed-by"
```

Comment:
move label keys to `internal/constant/const.go`

Reply:
done: [edc2a4a](https://github.com/apecloud/kubeblocks/pull/3733/commits/edc2a4a2e8d56bbf4af3f9425b905a06f60342ec)